infoblox_client/objects.py

```diff
@@ -448,12 +448,13 @@ def get_v6_class(cls):
 
 class Network(InfobloxObject):
     _fields = ['network_view', 'network', 'template',
-               'options', 'members', 'extattrs', 'comment']
+               'options', 'members', 'extattrs', 'comment',
+               'zone_associations']
```

Comment:
the basic version of network object donot include zone_associations field. hence, it should not be included in Network class as basic fields

Reply:
@adsri , I think there's a misunderstanding here.

What I was trying to say in my comment above (https://github.com/infobloxopen/infoblox-client/pull/187#issuecomment-369620183) is that, non-basic attributes should not be included in `_return_fields`.

We must, however, still include those non-basic attributes in `_fields` - otherwise, you cannot specify those attributes (to override default `_return_fields`) AT ALL when doing get/search.

Hope I make my clear this time.